LineStream/XargsStream.ts

```diff
@@ -0,0 +1,134 @@
+import { CommandBuilder, TextLineStream, toTransformStream } from "../deps.ts";
+import { LineStream, XargsFunction } from "./LineStream.ts";
+import {
+  ApplyFunction,
+  FilterFunction,
+  MapFunction,
+} from "../LineStream/Transformer.ts";
+
+export class XargsStream {
```

Comment:
can we make this inherit from a common class as LineStream, or make both classes implement a common interface?

so in the future the code stay more maintainable, if we add a new method to one we get a warning or an error to add the method to the other

this can be left for another pr, since this one is already is good

Reply:
Even at now, all that needs to be done is to change the LineStream implementation. 
I think it is fine as it is to be careful not to inadvertently overwrite CommandBuilder lines, text, etc.